src/layers/core/arc-layer/arc-layer.js

```diff
@@ -32,13 +32,12 @@ const defaultProps = {
   getTargetPosition: x => x.targetPosition,
   getSourceColor: x => x.color || DEFAULT_COLOR,
   getTargetColor: x => x.color || DEFAULT_COLOR,
-  strokeWidth: 1,
-  fp64: false
+  strokeWidth: 1
 };
 
 export default class ArcLayer extends Layer {
   getShaders() {
-    return this.props.fp64 ? {
+    return this.props.fp64 && this.props.projectionMode === 1 ? {
```

Comment:
64 only supports one projection mode?If so should we throw an error or a warning if other combinations are applied? This would just silently not render, right?